xml/cap_admin_certificates.xml

```diff
@@ -209,40 +209,75 @@
   <title>Rotating Automatically Generated Secrets</title>
 
   <para>
-   &cap; uses a number of automatically generated secrets for use internally.
-   These secrets have a default expiration of 10950 days and are set through
-   the <literal>CERT_EXPIRATION</literal> property in the
-   <literal>env:</literal> section of the <filename>scf-config-values.yaml</filename>
-   file. If rotation of the secrets is required, increment the value of
-   <literal>secrets_generation_counter</literal> in the
-   <literal>kube:</literal> section of the <filename>scf-config-values.yaml</filename>
-   configuration file (for example the example
-   <filename>scf-config-values.yaml</filename> used in this guide) then run
-   <command>helm upgrade</command>.
-  </para>
+   &cap; uses a number of automatically generated secrets (passwords and
+   certificates) for use internally provided by &operator;. This removes
+   the burden from human operators while allowing for secure communication.
 
-  <para>
-   This example demonstrates rotating the secrets of the <literal>scf</literal>
-   deployment.
-  </para>
-
-  <para>
-   First, update the <filename>scf-config-values.yaml</filename> file.
-  </para>
-
-<screen>kube:
-  # Increment this counter to rotate all generated secrets
-  secrets_generation_counter: 2</screen>
-
-  <para>
-   Next, perform a <command>helm upgrade</command> to apply the change.
+   From time to time, operators may wish to change such secrets, either manually
+   or on a schedule. This is called rotating a secret.
   </para>
+  <sect2 xml:id="sec-cap-secrets-rotation-finding">
+    <title>Finding secrets</title>
+    <para>
+     Retrieve the list of all secrets maintained by &kubecf;:
+    </para>
+<screen>
+&prompt.user;kubectl get quarkssecret --namespace kubecf
+    </screen>
+    <para>
+      To see information about a specific secret, for example the NATS
+      password:
+    </para>
+<screen>
+&prompt.user;kubectl get quarkssecret --namespace kubecf kubecf.var-nats-password --output yaml
+   </screen>
+   <para>
+     Note that each quarkssecret has a corresponding regular &kube; secret
+     that it controls:
+   </para>
+<screen>
+&prompt.user;kubectl get secret --namespace kubecf
+&prompt.user;kubectl get secret --namespace kubecf kubecf.var-nats-password --output yaml
+   </screen>
+  </sect2>
+  <sect2 xml:id="sec-cap-secrets-rotation-specific">
+    <title>Rotating specific secret</title>
```

Comment:
Think this should be either "Rotating a Specific Secret" or "Rotating Specific Secrets"